internal/builtinchecks/yamls/ssh-port.yaml

```diff
@@ -0,0 +1,10 @@
+name: "ssh-port"
+description: "Alert on deployments with containers exposing SSH port 22"
+remediation: "Don't expose SSH port as it's not recommended as per security best practice."
```

Comment:
let's be a bit more concrete with the remediation, something like: `Ensure that non-SSH services are not using port 22. Ensure that any actual SSH servers have been vetted.`